src/tic_tac_toe/game.clj

```diff
@@ -1,20 +1,42 @@
 (ns tic-tac-toe.game
   (:require [tic-tac-toe.board :refer :all]
             [tic-tac-toe.human-player :refer :all]
+            [tic-tac-toe.marks :refer :all]
             [tic-tac-toe.messages :refer :all]))
 
 (defn start-message
   []
   (do 
     (println welcome-message)
-    (present-board 3 initial-board)))
+    (present-board (make-initial-board 3))))
+
+(defn switch-player
+  [mark]
+  (if (= mark player-one-mark) player-two-mark
+    player-one-mark))
 
 (defn make-move
+  [current-board current-player]
+  (place-mark current-board (- (get-tile-number current-board) 1) current-player))
+
+(defn play-all-turns
   []
-  (present-board 3 (place-mark (- (get-tile-number) 1) "X")))
+  (loop [current-board (make-initial-board 3) 
+         current-player player-two-mark]
+    (cond 
+      (winner? current-board) (println (winner-message current-player))
+      (is-full? current-board) (println draw-message)
+      :else (do 
+              (let [current-player (switch-player current-player)
```

Comment:
Just a minor comment/suggestion: you could extract the lines of code for the ongoing game into a separate method to keep this method smaller and more readable, maybe? That's what I'd do cause I'm a nazi-methods-length...